app/controllers/admin/users_controller.rb

```diff
@@ -1,22 +1,17 @@
 class Admin::UsersController < Admin::BaseController
   before_action :load_user, only: %i[show edit update]
-  layout :get_layout
+  layout "design_system"
 
   def index
     @users = User.enabled.includes(organisation: [:translations]).sort_by { |u| u.fuzzy_last_name.downcase }
-    render_design_system(:index, :legacy_index)
   end
 
-  def show
-    render_design_system(:show, :legacy_show)
-  end
+  def show; end
 
   def edit
     unless @user.editable_by?(current_user)
       head :forbidden
-      return
```

Comment:
I believe you might still need a return here otherwise it will render the edit page? 
Can try it out?

Reply:
Hi Minno,
Ideally Edit link should not appear if user is not having GDS editor permission.
I will check again

Reply:
Hey Farah - the link would not appear, but if you visit the URL, i think the page will still render unless we have this return line